src/core_modules/capture-core/components/Pages/New/RegistrationDataEntry/RegistrationDataEntry.component.js

```diff
@@ -78,7 +129,7 @@ const RegistrationDataEntryPlain = (
                                     selectedScopeId={selectedScopeId}
                                     enrollmentMetadata={registrationMetaData}
                                     saveButtonText={'Save new'}
-                                    onSave={onSaveWithEnrollment}
+                                    onSave={() => handleSaveAttempt(onSaveWithEnrollment)}
                                     onGetUnsavedAttributeValues={() => console.log('onGetUnsavedAttributeValues will be here in the future in the future')}
```

Comment:
@JoakimSM I am not sure where I need those actions... You might remember? I havent really looked into it yet but if you dont remember no worries I can investigate.